Source/IntegrationTests/TestFiles/LitTests/LitTest/proof-obligation-desc/modify-frame-subset.dfy.expect

```diff
@@ -0,0 +1,8 @@
+modify-frame-subset.dfy(15,2): Error: modify statement might violate context's modifies clause
+ Asserted expression: (s[0] == s[3] || s[0] in {s[4]} || s[0] in {s[5]}) && (forall obj: C | obj in {s[1]} :: obj == s[3] || obj in {s[4]}) && forall obj: C | obj in {s[2]} :: obj == s[3] || obj in {s[5]}
```

Comment:
I think things like this are going to be super useful! With more complicated code, constructing the equivalent of this expression in your head can really be a pain.